extensions/cornerstone/src/commandsModule.js

```diff
@@ -334,24 +330,17 @@ const commandsModule = ({ servicesManager }) => {
       storeContexts: ['viewports'],
       options: {},
     },
-    // TODO: First/Last image
-    // Next/Previous series/DisplaySet
-    nextViewportDisplaySet: {
-      commandFn: actions.updateViewportDisplaySet,
-      storeContexts: [],
-      options: { direction: 1 },
-    },
-    previousViewportDisplaySet: {
-      commandFn: actions.updateViewportDisplaySet,
-      storeContexts: [],
-      options: { direction: -1 },
-    },
     // TOOLS
     setToolActive: {
       commandFn: actions.setToolActive,
       storeContexts: [],
       options: {},
     },
+    setZoomTool: {
+      commandFn: actions.setToolActive,
```

Comment:
I think we can use the setToolActive in this case or we would go the path of having one command per tool

Reply:
As `setZoomTool` is a hotkey command, hotkeys implementation does not support to runa command with options (yet!)

Reply:
I think we can keep this for now, but then lets create a follow-up issue where we allow hotkey definitions to include "options" that are passed to `runCommand`

Reply:
I've create an issue: #1447